index.html

```diff
@@ -40,9 +48,13 @@ <h3 class="presentation">I'm a Software Developer</h3>
     </ul>
   </section>
   <section class="works">
-    <h1 class="works_title">My Recent Works</h2>
-      <div class="project1">
-        <img src="./images/Yoga.svg" alt="yoga pose" class="arrow"></img>
+    <div class="works_div">
+      <h1 class="works_title">My Recent Works</h2>
+      <hr class="works_break">
+    </div>  
+    <div class="project1">
+      <img src="./images/Yoga.svg" alt="yoga pose" class="arrow"></img>
+      <div class="project1_div">
         <h2 class="project_title1">Multi-Post Stories</h3>
           <p class="project_description1">A daily selection of privately personalized reads; no accounts or sign-ups
```

Comment:
- [ ] I want to kindly suggest you take a look at the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=58J3PbIYgULPB2GD-0) and adjus your designs for this section to match it correctly.It is important that your design matches correctly the provided figma template as this shows your attention to details as a frontend developer and this is a good skill to have :+1: :sparkles: 
- figma design
![te](https://github.com/ommarz/Personal_Portfolio/assets/89055502/c56d70ee-5cfd-43af-9c36-884afeaa1f2f)
- your design
![fe](https://github.com/ommarz/Personal_Portfolio/assets/89055502/30380e71-5949-4b15-8bac-21e8df4134a6)